docs/getting_started/configuration2.md

```diff
@@ -0,0 +1,94 @@
+# Customizing Pelorus
```

Comment:
Reading the documentation, I though would be better to not split configuration in 2 different locations. I would merge this with https://pelorus.readthedocs.io/en/latest/Configuration/ . What you think?